lib/src/types/index.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for Pluggy
+// Definitions by: Pluggy <https://github.com/pluggyai>
+
+export * from './common'
+export * from './auth'
+export * from './connector'
+export * from './category'
+export * from './item'
+export * from './account'
+export * from './investment'
+export * from './transaction'
+export * from './identity'
+export * from './webhook'
```

Comment:
Nice job! Is this `./src/types` exported in the main index.ts file as well? So the client can do:
```
import { Email } from 'pluggy-node'
```
instead of 
```
import { Email } from 'pluggy-node/dist/types'
```

Reply:
Updated.